packages/vm/src/dummy_sha2.rs

```diff
@@ -0,0 +1,80 @@
+//! Dummy Sha-256 and Sha-224 Digest impls.
+//! These digests store/accept an already hashed value of the proper length.
+//! Just to comply with the Digest contract.
+//!
+//! Adapted from `sha2` [sha256.rs](https://github.com/RustCrypto/hashes/blob/master/sha2/src/sha256.rs)
+use sha2::digest::consts::{U28, U32};
+use sha2::digest::generic_array::GenericArray;
+use sha2::digest::{FixedOutputDirty, Reset, Update};
+
+use sha2::digest;
+
+/// The SHA-256 container
+#[derive(Clone)]
+pub struct Sha256 {
```

Comment:
Can we call this `Identity`, to make clear this is an identity hasher with input = output?

Reply:
OK. I'll call it `IdentitySha256` to differentiate it from `IdentitySha224` below.

Reply:
I think `IdentitySha256` is a better name, even after removing `Sha224`.

Reply:
Why do you think so? It is a `Digest` with 32 byte output that has no connection to SHA-2.

Reply:
You're right. I guess I was still thinking in terms of SHA-256 vs. SHA-224.

`Identity256`?

Reply:
> `Identity256`?

Sounds good